packages/commercetools/api-client/src/index.ts

```diff
@@ -2,5 +2,6 @@ export * from './fragments';
 export * from './types/Api';
 export * from './types/GraphQL';
 export * from './types/setup';
-export * from './helpers/commercetoolsLink/index';
+export { isAnonymousSession, isUserSession } from './helpers/utils';
+export { createErrorHandler } from './helpers/commercetoolsLink/graphqlError';
```

Comment:
`createCommerceToolsConnection` is missing, so it's a breaking change. If there is no reason for its removal, then please export it as well.

Reply:
It's being used directly in the index.server.ts

packages/commercetools/api-client/src/index.server.ts:6

Reply:
Currently, this package exports `createCommerceToolsConnection`, and after these changes, it no longer will.

Reply:
The exporting of `createCommerceToolsConnection` was one of the problems.

Because no package we are using use it and we were exporting it also.

Reply:
Updated the changelog, informing the breaking change

Reply:
> Because no package we are using use it and we were exporting it also.

It's true, but someone else might be using it (community or customer). Is there a particular reason why we should no longer export it?

Reply:
When you export it, you add to the nuxt bundle the apollo and commercetools SDK :(